lib/project_types/extension/models/specification_handlers/checkout_ui_extension.rb

```diff
@@ -4,12 +4,23 @@ module Extension
   module Models
     module SpecificationHandlers
       class CheckoutUiExtension < Default
+        L10N_FILE_SIZE_LIMIT = 16 * 1024 # 16kb
+        L10N_BUNDLE_SIZE_LIMIT = 256 * 1024 # 256kb
```

Comment:
Why do we need to validate individual file size? Isn't it enough to validate the bundle (all files)?

Reply:
@kumar303 @ecbrodie 
I placed these as pretty arbitrary numbers for now to trigger the discussion about choosing an appropriate size limit. I know we wanted to start out on the low side but we never came up with a number.

The limits placed here are meant to be permanent and are intended to be a first check/guard against submissions of very large payloads. We would also have a size check in the backend.

> Why do we need to validate individual file size
It may not be strictly necessary, but perhaps we can further nudge translation dictionaries towards being reasonable in size in doing so. I wanted to discuss the limits in general, like I mentioned.

Reply:
Also I meant permanent as in we would want to leave that code here, though in terms of what the actual limits are, we can adjust as needed. More easily by starting small and increasing if need be.